src/app/test/db/mongo/pruner_test.py

```diff
@@ -36,6 +38,60 @@ def test_prune_something(self, pruner, collection_mock):
         pruner.run()
         assert collection_mock.objects.return_value.no_cache.return_value.delete.called
 
+    def test_in_progress_ttl_index(self):
+        timeout = config._DB_SPEC["items"]["ttl"]["items"]["in_progress"].get("default")
+        test_pruner = MongoPruner(tasks=[task], ttl=timeout)
+        assert test_pruner._ttl == 15
+
+    def test_created_to_canceled(self, pruner):
+        test = Request(
```

Comment:
We do have some safeguards to cleanup test data, but you should still explicitly clean it up in your individual tests.  Instead of saving the Request here in your test, you'll want to create a fixture and call that.  Done properly, the fixture will clean up your data for you when the test is done.  There are lots of examples of this in other tests.